app/models/access_control_step.rb

```diff
@@ -95,8 +95,11 @@ def execute context
       lease.destroy
     end
     unless limited_access_submit
+      iso_duration = "P#{context[:add_lending_period_days]}DT#{context[:add_lending_period_hours]}H"
+      int_duration = ActiveSupport::Duration.parse(iso_duration).to_i
```

Comment:
I think it makes sense to do the same building method here as in the admin collections controller.

Reply:
Yep, definitely. I meant to switch it but got distracted by other changes and pushed before taking care of it. Good catch.